models/comments.models.js

```diff
@@ -14,4 +15,21 @@ function fetchComments(id) {
     })
 }
 
-module.exports = {fetchComments}
\ No newline at end of file
+function insertComment(newComment, article_id) {
+    if (Object.keys(newComment).length === 0 || newComment.body.length === 0) {
+        return Promise.reject({status:400, message: "Bad request"})
+    }
+    let sqlString =
+    `INSERT INTO comments (author, body, article_id)
+    VALUES %L
+    RETURNING *`
+
+    const formattedValues = [[newComment.username, newComment.body, article_id.article_id]]
+
+    const queryString = format(sqlString, formattedValues)
```

Comment:
Shouldn't need pg-format here as its only the one insertion. $1, $2, $3 should suffice 🙂